src/pfe/file-watcher/server/src/utils/kubeutil.ts

```diff
@@ -115,14 +122,25 @@ export async function getApplicationContainerInfo(projectInfo: ProjectInfo, oper
         const internalPorts: Array<string> = [];
         const exposedPorts: Array<string> = [];
         const appPorts = projectInfo.appPorts;
-        const resp = await k8sClient.api.v1.namespaces(KUBE_NAMESPACE).services.get({ qs: { labelSelector: releaseLabel } });
+        let resp: any = undefined;
+        let deploymentconfig: string = undefined;
+
+        if (projectInfo.projectType == "odo") {
```

Comment:
Why do we need to do this? Does Appsody has equivalent? We should try to avoid burning extension specific code into PFE. Can call into extension code for that?